core/src/main/java/software/amazon/awssdk/core/auth/EC2CredentialsProvider.java

```diff
@@ -58,9 +57,8 @@
 
     private RefreshResult<AwsCredentials> refreshCredentials() {
         try {
-            String credentialsResponse = EC2CredentialsUtils.getInstance()
-                                                            .readResource(credentialsEndpointProvider.getCredentialsEndpoint(),
-                                                                          credentialsEndpointProvider.getRetryPolicy());
+            String credentialsResponse = HttpCredentialsUtils.getInstance()
```

Comment:
did we end up deciding on the `create` vs `instance` vs `getInstance` naming? From what I remember, public things are always `create` and customers would be expected to memoize if needed. Internally, we'd use `instance` to indicate singletons?

Reply:
Ah yes - this was before that decision and I didn't pick that up in the merge conflict - will do.